batch_import/camio_hooks.py

```diff
@@ -61,6 +78,39 @@ def set_hook_data(data_dict):
         CAMIO_SERVER_URL = "https://test.camio.com"
     Log.debug("setting camio_hooks data as:\n%r", CAMIO_PARAMS)
 
+def get_account_info():
+    """
+    this function takes an auth token and gathers both the device ID of their
```

Comment:
Using the previously defined OAuth token, this function gets both the device_id and ip_address of the Camio Box associated with the user's account.